### PR TITLE
Video descriptions

### PR DESCRIPTION
**Pull request by ayyazdaniaryan**
You want 3 descriptions, not 2 around a video. Update to what was done previously.

### CODE REVIEW

**Comment by vincentfretin** [comment]
I don't understand. Do we want this for thematic in phase 1? For phase 2, it's an idea, not a thematic. And I thought that a single rich text was enough, you can put your video wherever you want.

**Comment by vincentfretin** [comment]
Never mind my previous comment, it's the ticket DEVAS-666.

File: assembl/graphql/schema.py [hunk 1]
```diff
@@ -900,6 +897,12 @@ def mutate(root, args, context, info):
                     kwargs[
                         'video_description_bottom'] = video_description_bottom
 
+                video_description_side = langstring_from_input_entries(
+                    video.get('description_entries_side', None))
+                if video_description_bottom is not None:
```
Comment (vincentfretin) [inline assembl/graphql/schema.py]: bad copy paste, video_description_side you mean here?

File: assembl/models/thematic.py [hunk 2]
```diff
@@ -70,13 +73,21 @@ class Thematic(Idea):
...
+
     video_html_code = Column(UnicodeText)
 
     crud_permissions = CrudPermissions(P_ADMIN_DISC, P_READ)
 
 
 LangString.setup_ownership_load_event(Thematic,
-    ['video_title', 'video_description_top', 'video_description_bottom'])
+    ['video_title', 'video_description_top', 'video_description_bottom',
+     'video_description_bottom'])
```
Comment (vincentfretin) [inline assembl/models/thematic.py]: video_description_bottom twice, you meant video_description_side?

File: assembl/tests/fixtures/graphql.py [hunk 3]
```diff
@@ -114,6 +114,10 @@ def thematic_with_video_and_question(graphql_request):
                 {value:"Câlisse de tabarnak",
                  localeCode:"fr"},
             ],
+            descriptionEntriesSide:[
+                {value:"Putain",
```
Comment (vincentfretin) [inline assembl/tests/fixtures/graphql.py]: really? :)

Comment (ayyazdaniaryan) [inline assembl/tests/fixtures/graphql.py]: Hehe

File: assembl/graphql/schema.py (hunk 1 above)
Comment (ayyazdaniaryan) [inline assembl/graphql/schema.py]: Yup, bad copy/paste. Resolved

**Pull request merged**